### DISCUSSION TITLE
REST API for user login with Basic Auth

### DISCUSSION THREAD

**Discussion by usaaama2010** [zitadel/zitadel#4915] [Q&A] [answered]
Is there any REST API to login users using username/password and obtain auth token in response?

**Comment by hifabienne** [comment, marked as answer]
Hi @usamaa2010 
No at the moment we do not have this possiblity.
But we are currently discussing a login/register api.

You can find the discussion here: https://github.com/zitadel/zitadel/discussions/4193

**Reply by usaaama2010** [reply]
Thanks, @hifabienne , I will follow the discussion then. :)
